packages/openneuro-app/src/scripts/dataset/components/AnalyzeDropdown.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react"
+import { Dropdown } from "@openneuro/components/dropdown"
+import { Button } from "@openneuro/components/button"
+import styled from "@emotion/styled"
+
+export interface CloneDropdownProps {
+  datasetId: string
+  snapshotVersion: string
+}
+
+const AnalyzeDiv = styled.div`
+  padding-right: 1em;
+`
+
+export const AnalyzeDropdown: React.FC<CloneDropdownProps> = (
+  { datasetId, snapshotVersion },
+) => {
+  const brainlifeUrl =
+    `https://brainlife.io/openneuro/${datasetId}/${snapshotVersion}`
+  const nemarUrl =
+    `https://nemar.org/dataexplorer/detail?dataset_id=${datasetId}`
+  const cbrainUrl =
+    `https://portal.cbrain.mcgill.ca/openneuro/${datasetId}/versions/${snapshotVersion}`
+  return (
+    <AnalyzeDiv className="clone-dropdown">
+      <Dropdown
+        label={
+          <Button
+            className="clone-link"
+            primary={true}
+            size="small"
+            label="Analyze"
+          >
+            <i className="fas fa-caret-up"></i>
+            <i className="fas fa-caret-down"></i>
+          </Button>
+        }
+      >
+        <div className="dataset-git-access">
+          <span>
+            <h4>Analyze this dataset</h4>
+          </span>
+          <p>
+            Work with OpenNeuro datasets using any of these third-party tools.
+          </p>
+          <hr />
+          <img
+            src="https://brainlife.io/favicon.ico"
```

Comment:
Should we host our own copies, so they can rearrange their sites without breaking this?

Reply:
There's trade offs either way but I think it does make sense. I'll make that change.